demo/src/main/java/com/example/demo/models/Squad.java

```diff
@@ -1,24 +1,49 @@
 package com.example.demo.models;
 
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.Id;
-import javax.persistence.OneToMany;
+
+import javax.persistence.*;
+import java.util.ArrayList;
 import java.util.List;
 
+@EqualsAndHashCode
```

Comment:
I believe we have discussed that it would be better to use manual implementation of `hashCode`/`equals` methods

```java
    @Override
    public boolean equals(Object obj) {
        if (obj instanceof Squad squad) {
            return Objects.equals(squad.getSquadId(), this.squadId);
        }
        return false;
    }

    @Override
    public int hashCode() {
        return Objects.hash(this.squadId);
    }
```